servers/rendering/shader_language.cpp

```diff
@@ -11824,6 +11845,7 @@ Error ShaderLanguage::complete(const String &p_code, const ShaderCompileInfo &p_
 				if (completion_base_array) {
 					if (current_uniform_hint == ShaderNode::Uniform::HINT_NONE) {
 						options.push_back("source_color");
+						options.push_back("color_conversion_disabled");
```

Comment:
This is the completion for textures. So color_conversion_disabled shouldn't be included. 

It should be around line 11789